frontstage_api/decorators/api_error_handler.py

```
@@ -0,0 +1,16 @@
+from frontstage_api.exceptions.exceptions import ApiError
+
+
+def api_error_handler(f):
```

Why can't we use the flask api.errorhandler here?

---

It's a slightly odd scenario, we don't always want to just return the error json when we encounter an error, e.g. for the `/messages_list` if the unread message total fails we still want to return the message list. 
If we use the `api.errorhandler` method it returns the response from that as the overall response to the incoming request.
This decorator just returns the error json as the return from the function it wraps

---

We can still do these things in the api.errorhandler right?